com.unity.cinemachine/Samples~/2D Samples/CameraMagnets.unity

```diff
@@ -38,7 +38,7 @@ RenderSettings:
   m_ReflectionIntensity: 1
```

Comment:
This is nice.  It would get even better if you add an onscreen toggle to enable/disable the camera magnets (won't work un less you sync with main to get the target group fix).  Also the light should be in the scene, not in the room prefab.

Reply:
Another improvement would be to disable vertical axis movement on the player so we can't make him walk into or out of the screen